commands/utils/queue.js

```diff
@@ -0,0 +1,13 @@
+import { SlashCommandBuilder } from '@discordjs/builders';
+
+export default {
+  data: new SlashCommandBuilder()
+    .setName('queue')
+    .setDescription(
+      'For testing purposes, prints some user data. In the future will add a User to queue'
+    ),
+  async execute(interaction) {
+    console.log(interaction);
```

Comment:
Functionality Looks good, potentially print more to discord to inform user about the queue